scripts/build_pkg.sh

```diff
@@ -39,6 +39,7 @@ fi
 DEPS_INSTALL_DIR=$BUILD_DIR/$CROSS_COMPILE/deps/usr
 FWUP_INSTALL_DIR=$BUILD_DIR/$CROSS_COMPILE/fwup-staging/usr
 PKG_CONFIG_PATH=$DEPS_INSTALL_DIR/lib/pkgconfig
+export ChocolateyInstall=$DEPS_INSTALL_DIR/chocolatey
```

Comment:
Unfortunately, choco won't run without that set.   Want to me to move it line 120 to make that clearer/cleaner?